Woodoku/src/Entity/WoodokuBoard.py

```diff
@@ -12,17 +14,36 @@ class WoodokuBoardRepresentation:
     The first block on the bottom row is (n - 1, 0).
     The bottom-right block's coordinate is (n - 1, n - 1).
     """
-    def __init__(self):
-        pass
+    board: ndarray
+    length: int
+
+    def __init__(self, n: int):
+        self.length = n
+        self.board = np.zeros((self.length, self.length))
```

Comment:
Document what 0 and 1 means for the board in the docstring of the class
Don't think it matters too much, but a 2d array of bool might be preferable with less memory footprint.

Reply:
Agreed.

Reply:
I see, I will change it